### PR TITLE
[esxi-host-exporter] changed configmap mount

### PR DESCRIPTION
**Pull request by christopherhans**

### CODE REVIEW

File: prometheus-exporters/esxi-host-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
             - mountPath: /atlas
               name: atlas-output
               readOnly: true
-            - mountPath: /
+            - mountPath: /exporter/
```
Comment (viennaa) [inline prometheus-exporters/esxi-host-exporter/templates/deployment.yaml:53]:
```suggestion
            - mountPath: /exporter/esxi-exporter-config.yaml
```

**Review by viennaa — CHANGES_REQUESTED** [review]

File: prometheus-exporters/esxi-host-exporter/templates/deployment.yaml (hunk 1 above)
Comment (christopherhans) [reply prometheus-exporters/esxi-host-exporter/templates/deployment.yaml:53]: why is this necessary?

Comment (viennaa) [reply prometheus-exporters/esxi-host-exporter/templates/deployment.yaml:53]: It is indeed not needed, just make sure you are using `/exporter/config.yaml` as the file to read in the exporter.

**Review by viennaa — APPROVED** [review]

File: prometheus-exporters/esxi-host-exporter/templates/deployment.yaml (hunk 1 above)
Comment (christopherhans) [reply prometheus-exporters/esxi-host-exporter/templates/deployment.yaml:53]: yes thats the case:
https://github.com/sapcc/esxi-exporter/blob/main/modules/configuration/ExporterConfig.py#L24

Comment (viennaa) [reply prometheus-exporters/esxi-host-exporter/templates/deployment.yaml:53]: well, ok ^^

**Pull request merged**